src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -413,3 +486,39 @@ void _stdcall TerminalSetCursorVisible(void* terminal, const bool visible)
     const auto publicTerminal = static_cast<const HwndTerminal*>(terminal);
     publicTerminal->_terminal->SetCursorVisible(visible);
 }
+
+COORD HwndTerminal::GetFontSize()
+{
+    return _actualFont.GetSize();
+}
+
+RECT HwndTerminal::GetBounds()
+{
+    RECT windowRect;
+    GetWindowRect(_hwnd.get(), &windowRect);
+    return windowRect;
+}
+
+RECT HwndTerminal::GetPadding()
+{
+    return {
+        0,
+        0,
+        0,
+        0,
+    };

```

Comment:
I mean I get it but you could have done it in one line instead of 6. :P

```
return { 0 };
```

Reply:
Thanks! I am clearly not a C++ expert 😅